app/uk/gov/hmrc/individualsdetailsapi/connectors/IfConnector.scala

```diff
@@ -102,8 +102,16 @@ class IfConnector @Inject()(
       Future.failed(new InternalServerException("Something went wrong."))
     }
     case notFound: NotFoundException => {
-      auditHelper.auditIfApiFailure(correlationId, None, matchId, request, requestUrl, notFound.getMessage)
-      Future.successful(emptyResponse)
+      notFound.message.contains("PERSON_NOT_FOUND") match {
+        case true => {
+          auditHelper.auditIfApiFailure(correlationId, None, matchId, request, requestUrl, notFound.getMessage)
+          Future.successful(emptyResponse)
+        }
+        case _ => {
+          auditHelper.auditIfApiFailure(correlationId, None, matchId, request, requestUrl, notFound.getMessage)
```

Comment:
Duplicated AuditHelper calls. Please move above match as it will be called either way prventing duplication.

Reply:
Also means the case's can be on 1 line e.g. case true -> Future.successful(emptyResponse)